src/utils.ts

```diff
@@ -168,7 +168,8 @@ export async function getFilesFromSourceFile({
   return lines
 }
 
-export function tempfile(extension = ''): string {
+export async function tempfile(extension = ''): Promise<string> {
+  const tempDirectory = await realpath(tmpdir())
```

Comment:
🚫 **[eslint]** <[@typescript-eslint/await-thenable](https://typescript-eslint.io/rules/await-thenable)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected `await` of a non-Promise (non-"Thenable") value.